api/python/stat_all.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Place Statistics - All
+nav_order: 14
+parent: Python
+grand_parent: API
+---
+
+# Get a Collection of Statistical Data for Multitple Places
+
+## `datacommons.get_stat_all(places, stat_vars)`
+
+Returns a nested `dict` of all time series for [`places`](https://datacommons.org/browser/Place) and [`stat_vars`](https://datacommons.org/browser/StatisticalVariable).
+
+**Arguments**
+
+- `places (Iterable of str)`: The `dcid`s of the [`Place`](https://datacommons.org/browser/Place) to query for.
+
+- `stats_vars (Iterable of str)`: The `dcid`s of the
+  [`StatisticalVariable`](https://datacommons.org/browser/StatisticalVariable).
+
+**Returns**
+
+A nested `dict` mapping Places to StatisticalVariables and all available
+time series for each Place and StatisticalVariable pair.
+
+The top level `dict` key is the place dcid and the second level `dict` key is the
+statistical variable dcid, with the object being an array of time series object
+with the following fields
+
+- `val`: an object from date to statistical value.
+- `importName`: the import name of the observations.
+- `provenanceDomain`: the provenance domain of the observations.
```

Comment:
s/provenance/[`Provenance`](https://datacommons.org/browser/Provenance)

Reply:
same with measurement method, observation period, unit, and scalingFactor

Reply:
done